kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/KafkaProxyStatusFactory.java

```diff
@@ -18,14 +18,20 @@
 
 public class KafkaProxyStatusFactory extends StatusFactory<KafkaProxy> {
 
+    private int replicaCount;
+
     public KafkaProxyStatusFactory(Clock clock) {
         super(clock);
     }
 
+    public void withReplicaCount(int replicaCount) {
```

Comment:
A bit confusing with being called `with*()`, but not matching the pattern we get with the sundrio-generated builders. So maybe `return this;` rather than the `void` return type?

Reply:
I did as you suggested initially and then intellij complained about un-used return values, but I think your right the consistency is a better choice.

Reply:
Ended up removing this all togther